apiv6/examples/demos/retry_demo/main.go

```diff
@@ -43,7 +45,7 @@ func showRetryingOnMultipleCodes() error {
 	demos.Step("Create Kentik API v6 client")
 	c := kentikapi.NewClient(kentikapi.Config{
 		SyntheticsAPIURL: s.URL,
-		RetryCfg: kentikapi.RetryConfig{
+		RetryCfg: httputil.RetryConfig{
```

Comment:
The retry mechanism should be the same for both versions of the API. It is not clear to me whether it is.

Reply:
The retry mechanism is the same which is probably better noticeable in PR #61. I will make a change here that hopefully will tidy up this particular PR.